### PR TITLE
@dzucconi: Add linkedin icons

### PR DESCRIPTION
**Pull request by jtotoole**

### CODE REVIEW

File: output/Read Me.txt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 Open *demo.html* to see a list of all the glyphs in your font along with their codes/ligatures.
 
+To use the generated font in desktop programs, you can install the TTF font. In order to copy the character associated with each icon, refer to the text box at the bottom right corner of each glyph in demo.html. The character inside this text box may be invisible; but it can still be copied. See this guide for more info: https://icomoon.io/#docs/local-fonts
```
Comment (dzucconi) [inline output/Read Me.txt]: FWIW in actual use you'll never really use the characters but refer to the classes like `.icon-linkedin`

**Comment by dzucconi** [comment]
:+1:

File: output/Read Me.txt (hunk 1 above)
Comment (jtotoole) [inline output/Read Me.txt]: Word, yeah looks like this bit got auto-generated since last time

Comment (dzucconi) [inline output/Read Me.txt]: Oh weird, I thought this was the README. :p

**Pull request merged**